Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Expect that `yarn build` has already been run.
 
 
-FROM node:fermium-bullseye-slim
+FROM node:fermium
```

Comment:
Was this needed?

Reply:
No -- good catch that is not necessary anymore